net.sourceforge.Pdfedit.json

```diff
@@ -0,0 +1,173 @@
+{
+    "app-id": "net.sourceforge.Pdfedit",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```

Comment:
Update runtime 20.08

Reply:
Yup, should use 20.08.

Reply:
Done. Comment aside: I think you're getting this repeteadly in the reviews because the [freedesktop-sdk gitlab frontpage](https://gitlab.com/freedesktop-sdk/freedesktop-sdk) still mentions 19.08 in the instructions for installing the freedesktop sdk (or at least that's where I got it from). It may be worthwhile to update that instructions if there's any maintainer for that here.

Reply:
Thanks, submitted a PR to change it.